rm -rf build && git reset --hard
patch -Np1 -i "${mypatd:-"../"}/rclonebrowser.patch"
mkdir build && cd build && cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr ..
diff notes
git clone --branch master --depth 1 --recursive https://github.com/DinCahill/RcloneBrowser.git rclonebrowser
diff a/src/main_window.cpp b/src/main_window.cpp
index e41faf5..09da58c 100644
--- a/src/main_window.cpp	1969-12-31 17:00:00.000000000 -0700
+++ b/src/main_window.cpp	1969-12-31 17:00:00.000000000 -0700
@@ -370,7 +370,7 @@ void MainWindow::rcloneListRemotes()
     });
 
     UseRclonePassword(p);
-    p->start(GetRclone(), QStringList() << "listremotes" << GetRcloneConf() << "-l" << "--ask-password=false", QIODevice::ReadOnly);
+    p->start(GetRclone(), QStringList() << "listremotes" << GetRcloneConf() << "--long" << "--ask-password=false", QIODevice::ReadOnly);
 }
 
 bool MainWindow::getConfigPassword(QProcess* p)
